packages/react-components/src/components/styles/ParticipantItem.styles.ts

```diff
@@ -14,17 +14,18 @@ export const participantItemContainerStyle = mergeStyles({
 });
 
 export const menuButtonContainerStyle = mergeStyles({
-  root: {
-    '&:hover': { background: 'none' },
-    '&:active': { background: 'none' }
-  }
+  paddingTop: '0.25rem'
 });
```

Comment:
I don't think we use this style anymore so we can delete this. Other than that LGTM. Thanks for awesome refactor of ParticipantItem. Looks much simpler now 👍